content/posts/July-2020.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "July 2020"
+date: 2020-07-03
+draft: false
+---
+
+## Week 27 (June 29th - July 3rd)
+
+### packit
+
+* We introduced a new command `packit validate-config` which validates the contect of a package config
```

Comment:
```suggestion
* We introduced a new command `packit validate-config` to validate the content of a package configuration files
```

Reply:
I'd even say 

    `packit.yaml` in your project

Reply:
But we support also different names:

```python
CONFIG_FILE_NAMES = [
    ".packit.yaml",
    ".packit.yml",
    ".packit.json",
    "packit.yaml",
    "packit.yml",
    "packit.json",
]
```